solutions/agents/variables.tf

```
@@ -135,36 +135,36 @@ variable "log_analysis_agent_tolerations" {
 
 variable "cloud_monitoring_enabled" {
   type        = bool
-  description = "Deploy IBM Cloud Monitoring agent if set as true."
+  description = "When set to `true`, this variable deploys the IBM Cloud Monitoring agent."
   default     = true
 }
 
 variable "cloud_monitoring_access_key" {
   type        = string
-  description = "Access key used by the IBM Cloud Monitoring agent to communicate with the instance."
+  description = "The access key that is used by the IBM Cloud Monitoring agent to communicate with the instance."
   sensitive   = true
   default     = null
 }
 
 variable "cloud_monitoring_secret_name" {
   type        = string
-  description = "The name of the secret which will store the access key. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the secret that will store the access key. If a prefix input variable is passed, the secret name is prefixed to the value in the `<prefix>-value` format."
```

```suggestion
  description = "The name of the secret to store the access key. If a value is specified in the `prefix` input, the name is set as `<prefix>-name`."
```

- I try to stay away from future tense if I can.
- I think that we need to explain better where the prefix is coming from. There are several other variables that would need this treatment.

---

@SirSpidey proposed edit:

"The name of the secret that stores the access key. If the specified value uses the `prefix` input, the name is set as `<prefix>-name`."

@maheshwarishikha can you provide more details about where the prefix is coming from?

---

@acolleen The prefix value comes from the `prefix` variable, which is why I added "`prefix` input". Input is the name for variables in projects (at least it used to be -- maybe they're just variables now).